docs/source/release_notes.rst

```diff
@@ -3,9 +3,14 @@
 Release Notes
 -------------
 
+Breaking Changes
```

Comment:
Let's move this below the contributor list for the "Future Release" section 

We can also have the current warning about Rolling primitives be another bullet point in the breaking changes list instead of being a warning like it is now